src/fighter.rs

```diff
@@ -59,6 +58,8 @@ impl Default for Stats {
 }
 
 /// Turns a fighter stub data (loaded from the metadata) into a fully active fighter.
+///
+/// PS: The position of the fighter corresponds the bottom center of the sprite.
```

Comment:
:+1:

Reply:
Actually, that is wrong now since I reverted the Anchor.